app/graphql/index.js

```diff
@@ -2,16 +2,18 @@ const { makeExecutableSchema } = require('graphql-tools'),
   types = require('./types'),
   inputs = require('./inputs'),
   users = require('./users'),
-  healthCheck = require('./healthCheck');
+  healthCheck = require('./healthCheck'),
+  albums = require('./albums');
 
-const typeDefs = [types, inputs, ...users.schemas, ...healthCheck.schemas];
+const typeDefs = [types, inputs, ...users.schemas, ...healthCheck.schemas, ...albums.schemas];
```

Comment:
something we are missing in the bootstrap is to automatically load all the schemas to the typedefs in this file